pkg/vmprovider/providers/vsphere/virtualmachine/delete.go

```diff
@@ -23,7 +24,11 @@ func DeleteVirtualMachine(
 	// Only a powered off VM can be destroyed.
 	if state != types.VirtualMachinePowerStatePoweredOff {
 		vmCtx.Logger.Info("Powering off VM prior to destroy", "currentState", state)
-		if err := ChangePowerState(vmCtx, vcVM, types.VirtualMachinePowerStatePoweredOff); err != nil {
+		if err := ChangePowerState(
+			vmCtx,
+			vcVM,
+			types.VirtualMachinePowerStatePoweredOff,
+			vmCtx.VM.Spec.PowerOffMode == vmopv1.VirtualMachineShutdown); err != nil {
```

Comment:
I think is reasonable in the case of a volume that is going to be reused to let it sync.

Reply:
I am not sure I understand your comment. Is this about the default behavior? Again, I agree that in v1a2 it should default to graceful by default, but for v1a1 we should preserve the existing behavior.